web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -318,16 +374,19 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   openIgnoreHealthErrorDialog() {
-    this.dialogSettings = new DialogSettings('Ignore', `Ignore Health Check for ${this.selectedTablet.label}`, '',
+    this.dialogSettings = new DialogSettings('Ignore', `Ignore Health Check for ${this.selectedTablet.label}`,
+                                             `Make ${this.selectedTablet.label} ignore certain health errors that match a pattern.`,
                                              `There was a problem ignoring the Health Check for ${this.selectedTablet.label}:`);
     this.dialogSettings.setMessage(`Ignored ${this.selectedTablet.label}`);
-    let flags = new PingTabletFlags(this.selectedTablet.alias).flags;
+    let flags = new IgnoreHealthCheckFlags(this.selectedTablet.alias).flags;
     this.dialogContent = new DialogContent('tablet_alias', flags, {}, undefined, 'IgnoreHealthError');
     this.dialogSettings.toggleModal();
   }
 
   openDemoteMasterDialog() {
-    this.dialogSettings = new DialogSettings('Demote', `Demote ${this.selectedTablet.label}`, '',
+    this.dialogSettings = new DialogSettings('Demote', `Demote ${this.selectedTablet.label}`,
+                                             `Makes a master tablet read-only, waits for all transactions to finish, ` +
```

Comment:
Out of curiosity: This means at the end of the command there won't be a master?

Will the slaves be reconfigured automatically?

Should we mention this here?